wochenende_cleanup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+outDir=$1
+mkdir -p $outDir/stats
+mkdir -p $outDir/fastqc
```

Comment:
This all looks sensible. Just try to keep on top of the space used, regularly use the tool `ncdu` to check your folder sizes.